Junit/src/IPL.java

```diff
@@ -0,0 +1,364 @@
+import java.util.*;
+import java.io.*;
+import java.util.stream.*;
+
+
+class Player implements Comparable<Player>{
+
+    String name;
+    String team;
+    String role;
+    int matches;
+    int runs;
+    double average;
+    double strikeRate;
+    int wickets;
+
+    Player(String name,String team,String role,int matches,int runs,double average,double strikeRate,int wickets){
+        this.name = name;
+        this.team = team;
+        this.role = role;
+        this.matches = matches;
+        this.runs  =runs;
+        this.average = average;
+        this.strikeRate  = strikeRate;
+        this.wickets = wickets;
+    }
+
+
+
+    public String getName(){
+        return name;
+    }
+
+    public String getRole(){
+        return role;
+    }
+
+    public int getWicket(){
+        return wickets;
+    }
+
+    public String getTeam(){
+        return team;
+    }
+
+    public int getRuns(){
+        return runs;
+    }
+
+    public int compareTo(Player p){
+        return 0;
+    }
+
+    // for all rounders
+    public int cumulativeScore(){
```

Comment:
Since this method is only required for All rounders
it should have made sense to model your player class such that
you have an AllrounderPlayer which extends from the Player class
and implement this method only this the sub class.